lochness/__init__.py

```diff
@@ -96,8 +96,8 @@ def initialize_metadata(Lochness, args, multiple_site_in_a_repo) -> None:
 
         elif 'rpms' in args.input_sources:
             # metadata.csv
-            id_fieldname = 'chric_subject_id'
-            consent_fieldname = 'chric_consent_date'
+            id_fieldname = Lochness['RPMS_id_colname']
```

Comment:
`RPMS_id_colname` and `RPMS_consent_colname` are loaded from the `config.yml` to Lochness object.

Reply:
What about `REDCap_id_colname` then?

Reply:
We can also have the same structure for REDCap. I can work on the redcap update later when I work with Z next time.

Reply:
Sorry, I did not understand it right last time. Now I see that the `redcap` block is already up there:

![image](https://user-images.githubusercontent.com/35086881/126175072-3ad881da-cb4c-4c9c-a3eb-c222403f1909.png)

Reply:
No action is necessary!